tests_fmt/build_exec_test.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+cabal build > /dev/null
```

Comment:
A test script should not compile the compiler. How would you use this in a CI setting?

Reply:
We have removed this script.